src/module/Ui/templates/entity/view/course-page.twig

```diff
@@ -19,6 +19,8 @@
  # @license   http://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
  # @link      https://github.com/serlo-org/athene2 for the canonical source repository
  #}
+{# Course should be translated, too #}
+{% 'course' | trans %}
```

Comment:
Types got only translated in their respective `entity/view` template. Since we don't have a view for the course itself, I added a non-used string here.

Reply:
removed in #805, because it was a syntax error, breaking course pages (`do` missing) and I added translating the course in my previous commit here:
https://github.com/serlo-org/athene2/blob/f6dd95818149fd2ce1125a84d7873f819bd4d7ab/src/module/Normalizer/src/Normalizer/Adapter/EntityAdapter.php#L171